docs/extensions/tutorials/_posts/1970-01-07-ModifyingRootApp.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: doc
+permalink: /docs/extensions/tutorials/modifying-root-app
+title: Modifying Root App
+section: Tutorials
+---
+
+# Modifying Root App
```

Comment:
I wouldn't call it root app, because there isn't any other app - just app. A perhaps better title would be: `Modifying app files` or `Modifying native project files`.